modules/op-release-notes-1-19.adoc

```diff
@@ -0,0 +1,396 @@
+
+* With this update, FIPS compliance is enhanced by adding `disable_spire` and `disable_tls` build tags to the entrypoint command compilation process. This enhancement ensures cryptographic symbols are properly removed from the entrypoint binary to meet compliance requirements.
+
+* With this update, parameter naming conventions across the `StepActions` feature have been unified for consistency. The updated `git-clone` `StepActions` uses consistent casing with `cache-fetch` and `cache-upload` `StepActions`.
+
+* With this update, the `/ok-to-test` memory feature is disabled by default. This precaution helps mitigate the risk of malicious code execution within testing environments. # (Need to confirm if this is added in Google Docs)
+
+* With this update, support is provided for expanding dynamic variables from remote pipeline definitions. This enhancement improves pipeline composition capabilities, which enables additional flexible pipeline architectures.
+
+* With this update, the {tekton-results} Watcher retries reconciliation multiple times before removing the finalizer, as long as the `storedDeadline` duration has not been exceeded. This improves reliability by reducing the chances of missing `TaskRun` or `PipelineRun` storage.
+
+* With this update, {tekton-results} supports field-based filtering for `Get` and `List` requests on `TaskRuns` and `PipelineRun`. When filter fields are specified, only the requested data is returned, improving response times and reducing network load.
+
+* With this update, you no longer need to switch to Kubernetes APIs to manage `ManualApprovalGate` Tasks. You can now use the `opc` tool to `approve`, `reject`, or `inspect` gates directly with commands such as `opc mag approve --task <name>` or `opc mag list`.
+
+* With this update, the `--live` flag in the `opc version --live` command queries the cluster to display the running versions of Chains, Pipelines, and other Tekton controllers, not just the local binary versions. In the previous release, users had to manually inspect cluster resources to determine which controller versions were deployed. Now, that information is displayed by using this command.
+
+* With this update, the `opc version` command now displays the {tekton-results} API client library version in addition to {pac} and Tekton versions. Before this release, this command only displayed versions for `opc`, `pac`, and `tkn`. This enhancement helps operators to easily verify cross-component compatibility between CLI tools and the cluster.
+
+* With this update, the `remember-ok-to-test` value in {tekton-config} is set to `false` by default to reduce the risk of executing untrusted or malicious code in test environments.
+
+* With this update, {pac} now supports triggering `PipelineRun` in GitLab using GitOps style comments such as `/test` and `/retest` on pushed commits.
```

Comment:
I think this duplicates the item on line 94: 
> `* With this update, {pac} extends GitOps workflow support in GitLab by enabling GitOps comments on pushed commits. This enhancement allows users to receive pipeline status updates directly on commit events, in addition to merge requests. Previously, this feature was only available for GitHub.`

Reply:
Removed from the content as discussed.